app/models/ExportsDeclaration.scala

```diff
@@ -50,6 +50,8 @@ case class ExportsDeclaration(
 
   def isComplete: Boolean = status == DeclarationStatus.COMPLETE
 
+  def capitalizedType: String = `type`.toString().toLowerCase().capitalize
```

Comment:
Seems a bit of a strange method to have on the model, Is it used in multiple places? if not lets just do this logic in the view/wherever

Reply:
I think we should avoid cluttering the models with helper methods that are only used in one place. It doesn't feel very Object Oriented.

Reply:
But on the other hand it hide internal object representation to "function" Contract allow to change `type` field without breaking external clients

Reply:
Is that really the best way to create a display text?  What if design aren't happy with "Simplified" or "Occasional" - then we'd have to change our enum definitions?

Reply:
Moved to the view